src/Actions/index.js

```diff
@@ -49,4 +49,8 @@ export const deletePalette = (paletteId, projectId) => ({
 export const deleteProject = (projectId) => ({
   type: 'DELETE_PROJECT',
   projectId
-});
\ No newline at end of file
+});
```

Comment:
Awesome! Glad we have the ability to expand the minipalette over to the larger palette view 🎨